templates/ai/index.html

```diff
@@ -133,16 +139,172 @@ <h2>Develop artificial intelligence projects on any environment</h2>
+            attrs={"class": "p-logo-section__logo"}
+            ) | safe
+          }}
+        </div>
+        <div class="p-logo-section__item">
+          {{
+            image (
+            url="https://assets.ubuntu.com/v1/8a6fdaab-opensearch-logo.png",
+            alt="",
+            width="319",
             height="256",
             hi_def=True,
             loading="lazy",
             attrs={"class": "p-logo-section__logo"}
             ) | safe
           }}
         </div>
+        <div class="p-logo-section__item">
+          {{
+            image (
+            url="https://assets.ubuntu.com/v1/f48de5d9-katib-logo.png",
+            alt="",
+            width="125",
+            height="384",
+            hi_def=True,
+            loading="lazy",
+            attrs={"class": "p-logo-section__logo"}
+            ) | safe
+          }}
+        </div>
+        <div class="p-logo-section__item">
+          {{
+            image (
+            url="https://assets.ubuntu.com/v1/54efd5a7-tensor-flow-logo.png",
+            alt="",
+            width="371",
+            height="384",
+            hi_def=True,
+            loading="lazy",
+            attrs={"class": "p-logo-section__logo"}
+            ) | safe
+          }}
+        </div>
+        <div class="p-logo-section__item">
+          {{
+            image (
+            url="https://assets.ubuntu.com/v1/5e048b8e-tensor-rt-logo.png",
+            alt="",
+            width="320",
+            height="384",
+            hi_def=True,
+            loading="lazy",
+            attrs={"class": "p-logo-section__logo"}
+            ) | safe
+          }}
+        </div>
+        <div class="p-logo-section__item">
+          {{
+            image (
+            url="https://assets.ubuntu.com/v1/795f2fc2-knative-logo.png",
+            alt="",
+            width="153",
+            height="384",
+            hi_def=True,
+            loading="lazy",
+            attrs={"class": "p-logo-section__logo"}
+            ) | safe
+          }}
+        </div>
+        <div class="p-logo-section__item">
+          {{
+            image (
+            url="https://assets.ubuntu.com/v1/20a0d896-pytorch-logo.png",
+            alt="",
+            width="336",
+            height="384",
+            hi_def=True,
+            loading="lazy",
+            attrs={"class": "p-logo-section__logo"}
+            ) | safe
+          }}
+        </div>
+        <div class="p-logo-section__item">
+          {{
+            image (
+            url="https://assets.ubuntu.com/v1/86e6c732-seldon-logo.png",
+            alt="",
+            width="374",
+            height="384",
+            hi_def=True,
+            loading="lazy",
+            attrs={"class": "p-logo-section__logo"}
+            ) | safe
+          }}
+        </div>
+        <div class="p-logo-section__item">
+          {{
+            image (
+            url="https://assets.ubuntu.com/v1/c6a757b1-pachyderm-logo.png",
```

Comment:
This one looks squished 
![Screenshot from 2023-11-17 12-16-14](https://github.com/canonical/ubuntu.com/assets/17607612/99b80dd9-fb62-4814-8830-4c8e2ef54a7e)